### ISSUE TITLE
https://prestigemad.com/ : Syntax error

### ISSUE THREAD

**Issue by dominiquefournier** [sharat87/prestige#12]
Hi
When I try to connect to https://prestigemad.com/ with Firefox, I have the message :
`Uncaught SyntaxError: expected expression, got '=' index.b332b9df.js:1:409888`
It stay blocked on "Loading..." message

**Comment by sharat87** [comment]
Oh hey, thanks for reporting this. I can't seem to reproduce this though. Can you try this out in another browser and see if you get the same error?

**Comment by dominiquefournier** [comment]
I try to disable all the extensions, but the problem persists on my Firefox 78.15.0esr (64 bits) on Debian (I disable the cache too, check if there is network error). 
If I try with Chromium, and the site works well.
Do you have another idea ?

**Comment by sharat87** [comment]
Ah I see. I didn't take the version number from your comment seriously at first glance, but it looks to be quite old. I am running Firefox 93 myself. I'm thinking that's the problem here. Is that the default version in Debian's repositories? If it is, I should then consider supporting it. Additionally, is it possible to upgrade your Firefox version and try with that to see if the problem goes away?

**Comment by dominiquefournier** [comment]
Yes it is the Debian Stable "Bullseye" Firefox Version. I can not upgrade the version : my system stay in stable, sorry.
https://packages.debian.org/search?keywords=firefox&searchon=names&suite=stable&section=all

**Comment by sharat87** [comment]
Oh no problem, that's okay. I'll do a test and see what we need to get this running on Firefox 78 at least. Thanks for reporting this. Please consider using Prestige on another browser like Chromium until I get this fixed. Thanks 🙏

**Comment by MattTennison** [comment]
Hey @sharat87, I was looking through my GitHub history and came back across this project.

For this - I suspect setting a browserlist as per https://parceljs.org/languages/javascript/#browser-compatibility would fix it. Happy to raise a PR if you'd like?

**Comment by sharat87** [comment]
Hey @MattTennison, thank you for reaching out. I've actually already made that change, see

https://github.com/sharat87/prestige/blob/0b60227cc58b831a73f5738b3e9cbcb898ad155f/frontend/package.json#L15-L20

But it still doesn't seem to work. I started Debian in a VM and opened Firefox 78 that comes shipped with it, and Prestige didn't work in it. I couldn't get back to take a closer look at what's happening there, but that's the status of this. I'd be more than happy to look into a PR to resolve this 😃.
